packages/cra-template/template/src/App.cy.jsx

```diff
@@ -1,8 +1,13 @@
 import App from './App';
 
 describe('Component tests', () => {
-  it('render Link', () => {
+  it('render Link component with text', () => {
     cy.mount(<App />);
     cy.get('[ui5-link]').should('be.visible').should('have.text', 'Getting Started with UI5 Web Component for React');
   });
+  it('type into Input component', () => {
+    cy.mount(<App />);
+    cy.get('[ui5-input]').typeIntoUi5Input('Hello there!');
+    cy.get('[ui5-label]').should('be.visible').should('have.text', 'Hello there!');
```

Comment:
```suggestion
    cy.get('[ui5-label]').should('be.visible').and('have.text', 'Hello there!');
```
Is more readable in my opinion